src/azure-cli/azure/cli/command_modules/profile/_help.py

```diff
@@ -95,9 +138,23 @@
     - name: Get an access token to use with MS Graph API
       text: >
         az account get-access-token --resource-type ms-graph
+    - name: >
+        Get an access token for a particular resource.  For a list of Virtual Network available service endpoints,
+        see `https://learn.microsoft.com/en-us/azure/virtual-network/virtual-network-service-endpoints-overview`.  
+        If you receive a `Failed to connect to MSI...` error, WHAT IS REALLY THE PROBLEM?  Is your resourceID wrong?  Does your resource not allow access tokens?
```

Comment:
What is the resource ID used? You may be hitting https://github.com/Azure/azure-cli/issues/11386.

Reply:
I was just testing with a random valid `resourceID` which begged the question, `which Azure resources have access tokens available?'